dist/tools/doccheck/exclude_simple

```diff
@@ -0,0 +1,8706 @@
+drivers/ir_nec/include/ir_nec_params.h:35: warning: Member IR_NEC_PARAM_PIN (macro definition) of file ir_nec_params.h is not documented.
+drivers/ir_nec/include/ir_nec_params.h:39: warning: Member IR_NEC_PARAMS (macro definition) of file ir_nec_params.h is not documented.
```

Comment:
These two actually need a regex to match the line numbers, otherwise anyone adding or deleting a line above the definition of the macros would need to touch this file again.

Reply:
right (they where like this in the exclude_patterns before)

Reply:
Oops :(

Care to piggy-back a fix?

Reply:
i made that a exclude simple line 
```
warning: Member IR_NEC_PARAM_PIN (macro definition) of file ir_nec_params.h is not documented.
warning: Member IR_NEC_PARAMS (macro definition) of file ir_nec_params.h is not documented.
```